dask/dataframe/io/parquet.py

```diff
@@ -94,9 +84,31 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None,
     else:
         index_col = None
 
+    # TODO: if categories vary from one rg to next, need to cope
```

Comment:
It it possible above to choose an index column that is not selected for loading?